### ISSUE TITLE
Does nothing once enabled.

### ISSUE THREAD

**Issue by lancekeltner7** [thebookisclosed/ShellFrosting#1]
This program, like Explorer Patcher works on 2 out of my 3 machines. On my work desktop, EP just loops the shell trying to load win11 first-run experience every 5-10sec.

With this program, the shell starts up right and doesn't loop. The extension shows disabled, but no changes are seen. Items are still all grouped, no matter the settings picked.

The only error I'm seeing in event log is:

Faulting application name: explorer.exe, version: 10.0.22621.1485, time stamp: 0x170d6771

Faulting module name: twinui.pcshell.dll, version: 10.0.22621.1485, time stamp: 0x19d32b65

Exception code: 0x80270233

Fault offset: 0x0000000000683990

Faulting process id: 0x0x5824

Faulting application start time: 0x0x1D9827410D95D3C

Faulting application path: C:\WINDOWS\explorer.exe

Faulting module path: C:\WINDOWS\system32\twinui.pcshell.dll

Report Id: 1ced8736-5a5c-4f7e-aa8b-c0ed4198aaee

**Comment by gus33000** [comment]
Hi,

as mentioned in the readme and the release notes (https://github.com/thebookisclosed/ShellFrosting#shell-frosting) the program is designed to allow you to experience unfinished features in Windows 11 Insider Builds of a specific range and specific branches only.

https://github.com/thebookisclosed/ShellFrosting/blob/main/README.md?plain=1#L6-L7

Your bug report indicates you're using ni_release Windows 11 builds (22621), which do not fall under the specific scope of this program.

In other words, it's not that it does not work, but rather it's not meant to be used on such builds and can't be made working on those either, as the goal here is to fix unfinished implementations that microsoft is currently working on in the shell on insider builds rather than giving you ungrouped taskbar items or labels (that's microsoft doing this, this program only helps you fix the work in progress code so it works and can be experienced)

**Comment by thebookisclosed** [comment]
It's a bit worrying how you've missed the clearly outlined supported builds explicitly mentioned
- [in the README](https://github.com/thebookisclosed/ShellFrosting#readme)
- [on the Releases page](https://github.com/thebookisclosed/ShellFrosting/releases)
- [in the Twitter post](https://twitter.com/thebookisclosed/status/1655686659436949507) (which publications often link to)

I get that this functionality is in high demand and it's disappointing if you don't get what you came here for — but if you've already made it as far as drafting a new issue, looking into obvious causes of what you're experiencing before pressing Submit would be nice.

Future versions might come with baked in version checks to prevent these kinds of issues getting opened, but so far I've deliberately avoided version checks to support eventual backports.
